src/torchcodec/decoders/_video_decoder.py

```diff
@@ -104,6 +109,7 @@ def __init__(
         num_ffmpeg_threads: int = 1,
         device: Optional[Union[str, torch_device]] = "cpu",
         seek_mode: Literal["exact", "approximate"] = "exact",
+        transforms: Optional[Sequence[DecoderTransform]] = None,
```

Comment:
On point 4:

> The type annotation that users will see only mentions accepting torchcodec.transforms.DecoderTransform. It does not mention the TorchVision transforms or nn.Module. The text of the docstring will say it, and I think that's enough?


One consequence of not accepting `nn.Module` is that  `mypy` and other type-checkers will complain when they see:

```py
VideoDecoder(..., transforms=[v2.Resize(...)])
```

which is a valid input.